stubs/Flask-JSONRPC/@tests/stubtest_allowlist.txt

```diff
@@ -0,0 +1,10 @@
+flask_jsonrpc.JSONRPCView.methods
```

Comment:
Can you comment on why these allowlists are necessary?

Reply:
It was in the previous release, and yesterday I published a new release.

Reply:
I see the "Third-party library removal policy" now, I understand.